### PR TITLE
Patch to copy command

### PR DESCRIPTION
**Pull request by jbobotek**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-c/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.
- I submitted this PR against the correct branch: 
  - [ ] This pull-request is submitted against the `master` branch. 
  - [ ] I have merged the latest `master` branch prior to submission and re-merged as needed after I took any feedback.
  - [ ] I have squashed my changes into one with a clear description of the change.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: build/.mxchip-ci.yml [hunk 1]
```diff
@@ -30,13 +30,13 @@ jobs:
...
+    name: $(MXCHIPAGENTPOOL)
   steps:
   - task: DownloadBuildArtifacts@0
     displayName: 'Download Build Artifacts'
     inputs:
       artifactName: 'drop'
 
   - script: |
-      cp $(System.ArtifactsDirectory)/drop/mxchip.bin /media/newt/AZ3166
+      echo $(NEWTIN) | sudo -S cp $(System.ArtifactsDirectory)/drop/mxchip.bin /media/newt/AZ3166
```
Comment (YoDaMa) [inline build/.mxchip-ci.yml]: for a multiline script you can just add the `sudo -S` line at the same indentation as echo. Makes for better looking code.

Also what is NEWTIN?

Comment (jbobotek) [reply build/.mxchip-ci.yml]: It's not multiline code, it's a pipe.

Comment (jbobotek) [reply build/.mxchip-ci.yml]: Printing again here, not a multiline script, it's a pipe.

**Pull request merged**